server/models/needModel.js

```diff
@@ -0,0 +1,8 @@
+// pgsql model for our group database
+const createTable = `CREATE TABLE IF NOT EXISTS Groups(
+  id SERIAL PRIMARY KEY,
```

Comment:
Is this supposed to be the same as the previous table? This might override groupModel.js create table

Reply:
Fixed!